nvflare/private/fed/client/client_executor.py

```diff
@@ -415,11 +386,11 @@ def abort_task(self, job_id):
 
     def _wait_child_process_finish(self, client, job_id, allocated_resource, token, resource_manager, workspace):
         self.logger.info(f"run ({job_id}): waiting for child worker process to finish.")
-        child_process = self.run_processes.get(job_id, {}).get(RunProcessKey.CHILD_PROCESS)
-        if child_process:
-            child_process.wait()
+        job_handle = self.run_processes.get(job_id, {}).get(RunProcessKey.JOB_HANDLE)
+        if job_handle:
+            job_handle.wait()
 
-            return_code = get_return_code(child_process, job_id, workspace, self.logger)
+            return_code = get_return_code(job_handle, job_id, workspace, self.logger)
 
             self.logger.info(f"run ({job_id}): child worker process finished with RC {return_code}")
             if return_code in [ProcessExitCode.UNSAFE_COMPONENT, ProcessExitCode.CONFIG_ERROR]:
```

Comment:
This comparison won't work unless return_code is standardized for all launchers.

Reply:
Added JobReturnCode(ProcessExitCode) to address this.